src/lib/viewers/doc/DocFindBar.js

```diff
@@ -93,9 +93,14 @@ class DocFindBar extends EventEmitter {
      * @return {void}
      */
     createFindButtons() {
-        const findPreviousButton = `<button class="bp-doc-find-prev" type="button">${ICON_FIND_DROP_UP}</button>`.trim();
-        const findNextButton = `<button class="bp-doc-find-next" type="button">${ICON_FIND_DROP_DOWN}</button>`.trim();
-        const findCloseButton = `<button class="bp-doc-find-close" type="button">${ICON_CLOSE}</button>`.trim();
+        const findPreviousLabel = __('find_previous');
+        const findPreviousButton = `<button class="bp-doc-find-prev" title="${findPreviousLabel}" type="button">${ICON_FIND_DROP_UP}</button>`.trim();
+
```

Comment:
What's the use of `trim()` when there are no whitespaces around? LGTM otherwise.

Reply:
Not sure, but I'm going to leave it as-is since it was already there.